kubernetes/07-mongodb-statefulset.yaml

```
@@ -29,7 +29,12 @@ spec:
             limits:
               memory: "512Mi"
               cpu: "500m"
-      volumes:
-        - name: mongodb-storage
-          persistentVolumeClaim:
-            claimName: mongodb-pvc
\ No newline at end of file
+  volumeClaimTemplates:
+      - metadata:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)